packages/core-modules/contracts/modules/CoreElectionModule.sol

```diff
@@ -73,29 +72,28 @@ contract CoreElectionModule is IElectionModule, ElectionStorage, OwnableMixin {
     }
 
     // TODO: add pagination for getting nominees list
-    function getNominees() external view override returns (address[] memory) {
-        return _currentElectionData().nominees;
+    function getCandidates() external view override returns (address[] memory) {
+        return _currentElectionData().candidates;
     }
 
     function nominate() external override {
         ElectionData storage electionData = _currentElectionData();
 
-        if (electionData.nomineePositions[msg.sender] != 0) {
+        if (electionData.candidatePositions[msg.sender] != 0) {
             revert AlreadyNominated(msg.sender);
         }
 
-        electionData.nominees.push(msg.sender);
-        electionData.nomineePositions[msg.sender] = electionData.nominees.length;
-        electionData.nomineeVotes[msg.sender] = 0;
```

Comment:
No need to set it to 0 since each election uses a fresh electionData and votes are scheduled after the nomination